### DISCUSSION TITLE
Why my answer getting so long

### DISCUSSION THREAD

**Discussion by thanhtantran** [imartinez/privateGPT#738] [General]
Hello,

I put an user manual as source document, and ask only a simple question, why the bot answer too long as needed

![Screenshot_1](https://github.com/imartinez/privateGPT/assets/5319910/34c85871-3561-4509-845e-4abeca0e5305)

**Comment by egorgutorovv** [comment]
Hi, how long did it take to handle your query?

**Reply by thanhtantran** [reply]
About 3 mins
